samples/military_feature_schemas/comparison_test_results/tested_Fields_Activities.csv

```diff
@@ -0,0 +1,29 @@
+field_name,field_type,field_length,field_alias,nullability,field_domain,field_default,field_notes,field_comparison_result
+identity,Integer,4,Standard Identity,NULLABLE,Identity,,,OK
+context,SmallInteger,2,Context,NULLABLE,Context,,,OK
+symbolset,SmallInteger,2,Symbol Set,NON_NULLABLE,Activities,40,,Activities ; 40
+entity,Integer,4,Activities Symbol,NON_NULLABLE,Activities_Entities,,,OK
+modifier1,Integer,4,Modifier 1,NULLABLE,Activities_Modifier_Ones,,,OK
+echelon,Integer,4,Echelon,NULLABLE,Echelons,,,OK
+indicator,Integer,4,"HQ, Task Force, Feint/Dummy",NULLABLE,HQTFFD,,,OK
+uniquedesignation,String,30,Unique Designation,NULLABLE,,,,OK
+additionalinformation,String,50,Additional Information,NULLABLE,,,,OK
+staffcomment,String,50,Staff Comments,NULLABLE,,,,OK
+reinforced,SmallInteger,2,Reinforced or Reduced,NULLABLE,Reinforced,,,OK
+combateffectiveness,String,5,Combat Effectiveness,NULLABLE,CombatEffectiveness,,,3
+credibility,SmallInteger,2,Credibility,NULLABLE,Credibility,,,OK
+reliability,String,2,Reliability,NULLABLE,Reliability,,,OK
+idmode,String,15,IFF/SIF/AIS,NULLABLE,,,,OK
+quantity,SmallInteger,2,Quantity,NULLABLE,,,,OK
+x,Double,8,X Location,NULLABLE,,,,OK
+y,Double,8,Y Location,NULLABLE,,,,OK
+z,Double,8,Altitude/Depth,NULLABLE,,,,Altitude or Depth
```

Comment:
Not yet, but I will once the review of these is completed and there are no obvious changes that need to be made.